pkg/api/defaults.go

```diff
@@ -70,6 +70,7 @@ func DefaultOperatorFlags() OperatorFlags {
 		"aro.banner.enabled":                       flagFalse,
 		"aro.checker.enabled":                      flagTrue,
 		"aro.dnsmasq.enabled":                      flagTrue,
+		"aro.restartdnsmasq.enabled":               flagFalse,
```

Comment:
Since our next PUCM will already cause reboots, we can merge with this enabled:
```suggestion
		"aro.restartdnsmasq.enabled":               flagTrue,
```